cloud-infrastructure/terraform/environments/development/aws/data/main.tf

```diff
@@ -119,6 +123,84 @@ module "hm_argo_cd_helm_chart" {
   ]
 }
 
+# Airbyte
+# Airbyte - S3 bucket
+module "hm_amazon_s3_bucket_hm_airbyte" {
+  source         = "../../../../modules/aws/hm_amazon_s3_bucket"
+  s3_bucket_name = "${var.environment}-hm-airbyte"
+  environment    = var.environment
+  team           = var.team
+}
+# Airbyte - IAM user
+module "hm_airbyte_iam_user" {
+  source            = "../../../../modules/aws/hm_airbyte_iam_user"
+  aws_iam_user_name = "${var.environment}_hm_airbyte_user"
+  s3_bucket_name    = module.hm_amazon_s3_bucket_hm_airbyte.name
+  environment       = var.environment
+  team              = var.team
+}
+# Airbyte - Postgres
+locals {
+  amazon_rds_name = "${var.environment}-hm-airbyte-postgres"
+}
+data "aws_secretsmanager_secret" "hm_airbyte_postgres_secret" {
+  name = "${var.environment}-hm-airbyte-postgres/admin"
+}
+data "aws_secretsmanager_secret_version" "hm_airbyte_postgres_secret_version" {
+  secret_id = data.aws_secretsmanager_secret.hm_airbyte_postgres_secret.id
+}
+module "hm_hm_airbyte_postgres_security_group" {
+  source                         = "../../../../modules/aws/hm_airbyte_postgres_security_group"
+  amazon_ec2_security_group_name = "${local.amazon_rds_name}-security-group"
+  amazon_vpc_id                  = data.aws_vpc.hm_amazon_vpc.id
+  environment                    = var.environment
+  team                           = var.team
+}
+module "hm_hm_airbyte_postgres_subnet_group" {
+  source            = "../../../../modules/aws/hm_amazon_rds_subnet_group"
+  subnet_group_name = "${local.amazon_rds_name}-subnet-group"
+  subnet_ids        = ["subnet-xxxxxxxxxxxxxxxxx", "subnet-xxxxxxxxxxxxxxxxx", "subnet-xxxxxxxxxxxxxxxxx", "subnet-xxxxxxxxxxxxxxxxx"]
+  environment       = var.environment
+  team              = var.team
+}
+module "hm_hm_airbyte_postgres_parameter_group" {
+  source               = "../../../../modules/aws/hm_amazon_rds_parameter_group"
+  family               = "postgres16"
+  parameter_group_name = "${local.amazon_rds_name}-parameter-group"
+  # https://stackoverflow.com/questions/78645095/pod-airbyte-temporal-failed-to-connect-to-rds-with-rds-force-ssl-enabled
+  parameters = [
+    {
+      name  = "rds.force_ssl"
+      value = "0"
```

Comment:
Unfortunately, I didn't find a perfect solution here. I posted all details I found at
- https://stackoverflow.com/questions/78645095/pod-airbyte-temporal-failed-to-connect-to-rds-with-rds-force-ssl-enabled
- https://github.com/airbytehq/airbyte/issues/39636